1-server_unit/README.md

```diff
@@ -12,7 +12,7 @@ This ansible is checking the operation with Personium version 1.5.2 later and Ce
 
 |**Servers**      |   **Role**      |   **MW**                           |  **default memory size** (*1) |   **AWS EC2 specs** (*2)|  
 |:----------------|:----------------|:-----------------------------------|:------------------------------|:-------------------------|
-| Server 1        |  Bastion,Web,AP,ES    | nginx,tomcat(768MB),memcached(256MB*2),Elasticsearch(1875MB)                     |3155MB                         |      m3.medium            |
+| Server 1        |  Bastion,Web,AP,ES,MQ,NFS    | nginx,tomcat(768MB),memcached(256MB*2),Elasticsearch(1875MB),ActiveMQ                     |3155MB                         |      m3.medium            |
```

Comment:
In order to match with 3server,
The order of Bastion, Web, AP, ES, MQ, NFS,
I think that the order of Bastion, Web, AP, NFS, MQ, ES is better